webapp/pom.xml

```diff
@@ -106,7 +106,7 @@
          <plugin>
              <groupId>org.eclipse.jetty</groupId>
              <artifactId>jetty-maven-plugin</artifactId>
-             <version>9.3.3.v20150827</version>
```

Comment:
Do you think you can look into removing the extra things in the logs when we upgrade jetty? 
This is the log for running the platform off the master branch (without any modules): https://pastebin.com/XJYNQuVH
And this one is after the jetty upgrade: https://pastebin.com/53CJ63iZ

Reply:
Could you try with the latest update? I'm not certain I have things setup correctly on my environment, but I think I should've fixed that.